assets/css/_map_page.scss

```
@@ -23,6 +23,10 @@ $map-page-search-panel-width: 24rem;
   margin: 1rem;
 }
 
+.c-map-page .c-route-properties-card {
+  margin: 1rem;
+}
+
```

suggestion(non-blocking): Thoughts on merging this with the style above this one?
(You could also do this via sass nesting)

```suggestion
.c-map-page :is(.c-vehicle-properties-card, .c-route-properties-card) {
  margin: 1rem;
}

```

---

f5c6edb3